pyproject.toml

```diff
@@ -244,7 +247,12 @@ black = "black wetterdienst example tests"
 isort = "isort wetterdienst example tests"
 format = ["black", "isort"]
 lint = "flakehell lint wetterdienst example tests"
-test = "pytest tests"
+docs = { shell = "cd docs && poetry run make html" }
+test = "pytest -vvv tests"
```

Comment:
what about 

test = ["test-parallel", "test-cflakes"]

Reply:
I would also be eager to phrase it like that. However, running the tests in parallel still does not work on CI, see my attempts 1644b7ede401 vs. 8e64673eed02. This has to be investigated further.